change/@fluentui-react-spinbutton-cffa2c98-0617-4b61-ac17-e7b6d96dccb1.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 28 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26867&commitId=4b439e6b7d7ed43778047ffc14689a5ee31d84f9&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1820 screenshots unchanged, 0 screenshots with different dimensions and 28 screenshots with visible difference.


  <details><summary><b>unknown</b> 28 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SpinButton Converged.Disabled - Dark Mode.focused.chromium.png|155| Changed|
|SpinButton Converged.Disabled - Dark Mode.hoverDecrement.chromium.png|155| Changed|
|SpinButton Converged.Disabled - Dark Mode.hoverIncrement.chromium.png|155| Changed|
|SpinButton Converged.Disabled - Dark Mode.hoverInput.chromium.png|155| Changed|
|SpinButton Converged.Disabled - Dark Mode.mouseDownDecrement.chromium.png|136| Changed|
|SpinButton Converged.Disabled - Dark Mode.mouseDownIncrement.chromium.png|136| Changed|
|SpinButton Converged.Disabled - Dark Mode.rest.chromium.png|126| Changed|
|SpinButton Converged.Disabled - High Contrast.focused.chromium.png|13| Changed|
|SpinButton Converged.Disabled - High Contrast.hoverDecrement.chromium.png|13| Changed|
|SpinButton Converged.Disabled - High Contrast.hoverIncrement.chromium.png|13| Changed|
|SpinButton Converged.Disabled - High Contrast.hoverInput.chromium.png|13| Changed|
|SpinButton Converged.Disabled - High Contrast.mouseDownDecrement.chromium.png|13| Changed|
|SpinButton Converged.Disabled - High Contrast.mouseDownIncrement.chromium.png|13| Changed|
|SpinButton Converged.Disabled - High Contrast.rest.chromium.png|162| Changed|
|SpinButton Converged.Disabled - RTL.focused.chromium.png|139| Changed|
|SpinButton Converged.Disabled - RTL.hoverDecrement.chromium.png|139| Changed|
|SpinButton Converged.Disabled - RTL.hoverIncrement.chromium.png|139| Changed|
|SpinButton Converged.Disabled - RTL.hoverInput.chromium.png|139| Changed|
|SpinButton Converged.Disabled - RTL.mouseDownDecrement.chromium.png|168| Changed|
|SpinButton Converged.Disabled - RTL.mouseDownIncrement.chromium.png|168| Changed|
|SpinButton Converged.Disabled - RTL.rest.chromium.png|120| Changed|
|SpinButton Converged.Disabled.focused.chromium.png|139| Changed|
|SpinButton Converged.Disabled.hoverDecrement.chromium.png|139| Changed|
|SpinButton Converged.Disabled.hoverIncrement.chromium.png|139| Changed|
|SpinButton Converged.Disabled.hoverInput.chromium.png|139| Changed|
|SpinButton Converged.Disabled.mouseDownDecrement.chromium.png|168| Changed|
|SpinButton Converged.Disabled.mouseDownIncrement.chromium.png|168| Changed|
|SpinButton Converged.Disabled.rest.chromium.png|120| Changed|

</details>


<div id="vrtCommentfluentuiv9"/>